examples/SampleApp/src/components/NetworkDownIndicator.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { Spinner, useTheme } from 'stream-chat-react-native';
+
+
+const styles = StyleSheet.create({
+  networkDownContainer: {
+    alignItems: 'center',
+    flexDirection: 'row',
+  },
+  networkDownText: {
+    marginLeft: 10,
+  },
+});
+
+export const NetworkDownIndicator: React.FC<{ titleSize: 'small' | 'large' }> = ({ titleSize = 'small' }) => {
```

Comment:
it looks like you're using this component once and passing in the titleSize as large from there. why not just default it to large if it's the only options that's used?